app/bundles/CampaignBundle/Command/TriggerCampaignCommand.php

```diff
@@ -167,6 +168,34 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $threadId      = $input->getOption('thread-id');
         $maxThreads    = $input->getOption('max-threads');
 
+        if (is_numeric($id)) {
+            $id = (int) $id;
+        }
+
+        if (is_numeric($maxThreads)) {
+            $maxThreads = (int) $maxThreads;
+        }
```

Comment:
As we need either `int` or `null` in the`ContactLimiter`, how about to do this?
```suggestion
        $maxThreads = is_numeric($maxThreads) ? (int) $maxThreads : null;
```

Reply:
I think the changes you propose should be a separate PR.
Every command should control the type of the option/argument with the appropriate message. For example if you set the `max-threads` option to _aaaa_ then the application should exit with an error code and an appropriate message.
Otherwise _some_ users could be confused why the application does not work as they would expect that `contact-id`, `min-contact-id`, etc. instead of giving an error is producing unexpected results converting the option is either a numeric or null.

Otherwise this PR should contain messages for affected commands, or extract command-related type changes into a separate PR and do it there.

Reply:
You are right. If they send `aaa` to an int param then they get an exception that somewhere in the code we expect an int and got an string. Better than silently resetting the param.